Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM  kalilinux/kali
```

Comment:
any reason why this should be a Kali image and not something slim that only has python? like `python:alpine` ?